docs/source/config/encoders/esjson.rst

```diff
@@ -36,7 +36,7 @@ Config:
     indexed into ElasticSearch. Available fields to choose are "Uuid",
     "Timestamp", "Type", "Logger", "Severity", "Payload", "EnvVersion", "Pid",
     "Hostname", and "Fields" (where "Fields" causes the inclusion of any and
```

Comment:
Other than changing "Fields" to "DynamicFields", the behaviour would be identical. So including "DynamicFields" in the list without setting a separate `dynamic_fields` setting would include all of them, just like it does now.

I'm okay w/ breaking the config, as long as we explicitly fail... i.e. Heka should fail to start if "Fields" (or anything other than the set of supported values, actually) is in the list. What I *don't* want to do is have Heka start up and silently do the wrong thing.